index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>Portfolio</title>
+    <title>Portfolio Project</title>
```

Comment:
- [ ] Please make sure your `<title>` adequately and briefly describes the content of the page this helps in easy navigation especially if the user has multiple tabs open in their browser 🔍.
    - Hint: It would be better if you put your name: `Kerlos Portfolio`